egs2/aphasiabank/asr1/run.sh

```diff
@@ -1,28 +1,34 @@
 #!/usr/bin/env bash
-# Set bash to 'debug' mode, it will exit on :
-# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+
+# Aphasia English E-Branchformer experiment
+
 set -e
 set -u
 set -o pipefail
 
+asr_tag="ebranchformer_wavlm"
+
 train_set="train"
 valid_set="val"
 test_sets="test"
 include_control=true
+tag_insertion=none
 
-asr_config=conf/tuning/train_asr_ebranchformer_small_wavlm_large.yaml
+asr_config=conf/tuning/train_asr_ebranchformer_small_wavlm_large1.yaml
```

Comment:
```suggestion
asr_config=conf/train_asr.yaml
```

Reply:
I specified the full path to the config file because the value of `feats_normalize` depends on whether WavLM is being used or not (https://github.com/espnet/espnet/pull/5104/files#diff-1aba163fe023c77a4da555552adb37dc70fb8de832562d4891075fe10c08cae1R20-R24).

This is also the reason that I removed `train_asr.yaml` since there is not way to tell if WavLM is used inside the config.

Do you have suggestions for a better way to do this?
One solution I have in mind is writing a very simple python script under `local/` which loads the yaml file and detect if SSLR is used and print true/false.
Then I can call it in `run.sh` and assign a suitable value to `feats_normalize` based on the output.